### PR TITLE
Improve chemical resistance patterns

### PR DESCRIPTION
**Pull request by jseager7**
This pull request fixes some incorrect or missing term IDs in the pattern 'abnormally increased resistance to chemical entity', and it also replaces the pattern 'abnormally _increased sensitivity_ to chemical entity' with the pattern 'abnormally _decreased resistance_ to chemical entity'. I think it makes more sense to have both these patterns as modifiers of one trait (resistance), rather than modifying two traits (resistance and sensitivity).

### CODE REVIEW

File: src/patterns/dosdp-workshop/abnormallyDecreasedResistanceToChemicalEntity.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+pattern_name: abnormallyDecreasedResistanceToChemicalEntity.yaml
+pattern_iri: http://purl.obolibrary.org/obo/upheno/patterns-dev/abnormallyDecreasedResistanceToChemicalEntity.yaml
+description: "A phenotype in which an organism shows decreased resistance to a chemical, for example 4-aminosalicylic acid."
```
Comment (matentzn) [inline src/patterns/dosdp-workshop/abnormallyDecreasedResistanceToChemicalEntity.yaml]: I would like the "/increased sensitity" synonymous phrase to appear in the definition. Objections? @dosumis @mah11

File: src/patterns/dosdp-workshop/abnormallyDecreasedResistanceToChemicalEntity.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  inheres_in_part_of: RO:0002314
+  has_modifier: RO:0002573
+  has_part: BFO:0000051
+
+annotationProperties:
+  exact_synonym: oio:hasExactSynonym
+
+vars:
+  chemical_entity: "'chemical_entity'"
+
```
Comment (matentzn) [inline src/patterns/dosdp-workshop/abnormallyDecreasedResistanceToChemicalEntity.yaml]: @mah11 should we include the increased sensitivity as an explicit synonym?

**Comment by jseager7** [comment]
@matentzn Apologies if you're already aware of this, but I've just realised that that this pull request clashes with #485, which opts for 'increased sensitivity' in response to 'increased resistance'. If we decide to include 'increased sensitivity' as a synonym, then this pull request could close that issue too.

**Comment by matentzn** [comment]
Lets wait for @chris-grove to be back and we will reconcile the two. You still prefer the resistance route over the sensitivity route?

**Comment by jseager7** [comment]
> You still prefer the resistance route over the sensitivity route?

Only because I'd prefer to have these patterns on one 'dimension' rather than two. I don't have a strong preference. I'm fine to be overruled if there are specific advantages to the resistance/sensitivity approach (such as it being more conventional to biologists).

I'm not suggesting this naming convention for PHIPO's benefit, since we're actually following the resistance/sensitivity approach in term naming, e.g. 'resistance to penicillin' / 'sensitivity to penicillin'. We can call the underlying patterns whatever we like, after all.

**Comment by chris-grove** [comment]
@jseager7 @matentzn 
I'm fine with going with "increased resistance" / "decreased resistance" in the logical definitions and pattern titles as long as the appropriate synonyms are in place, e.g. "decreased sensitivity" / "increased sensitivity", respectively, for the sake of end users browsing the ontology looking for the corresponding "sensitivity" terms.

**Comment by jseager7** [comment]
@chris-grove I've added 'increased sensitivity' and 'decreased sensitivity' as synonyms to their respective patterns (see https://github.com/obophenotype/upheno/pull/491/commits/da2fbfce3f41158c0edd80f0a5a4889372317acc). @matentzn I've copied the syntax from other patterns, so I assume it's correct, but I'd appreciate some double-checking (unless the CI pipeline will take care of this anyway).

**Comment by matentzn** [comment]
@jseager7 for increased and decreased, please use inheres in instead of inheres in part of! :) Thanks.

**Comment by jseager7** [comment]
@matentzn I've changed the relation to 'inheres_in' (see commit 564a55e).

**Comment by matentzn** [comment]
Thanks all for the hard work! Merging.

**Pull request merged**